alf/environments/carla_sensors.py

```diff
@@ -2050,3 +2050,74 @@ def _world_to_pixel(self, location: carla.Location, projective=False):
         For example: top leftmost location will be a pixel at (0, 0).
         """
         return self._map_handler.world_to_pixel(location, projective)
+
+
+# ==============================================================================
+# -- Obstacle Detection Sensor -------------------------------------------------
+# ==============================================================================
+
+
+@alf.configurable
+class ObstacleDetectionSensor(SensorBase):
+    """ObstacleDetectionSensor.
```

Comment:
I think it is still good to document arguments about `hit_radius` and `only_dynamics` as the name does not speak much about them.